plugins/modules/ovirt_vm.py

```diff
@@ -906,14 +906,23 @@
         type: str
     virtio_scsi_enabled:
         description:
-            - "Enable Virtio SCSI support."
+            - "If I(true), the Virtio-SCSI devices will obtain a number of multiple queues depending on
+               the available virtual Cpus and disks, or according to the specified virtio_scsi_multi_queues."
```

Comment:
AFAIR we don't enable the virtio-scsi multi queue by default when enabling virtio-scsi (adding the virtio-scsi controller) - did you check it is indeed enabled by default? if it's not the case, we shouldn't change this comment

Reply:
My bad, actually I got confused twice with those values

now I thought virtio_scsi_enabled is actually virtio_scsi_multi_queues_enabled, because of that I changed its description

I didn't realize until now that there is no virtio_scsi_multi_queues_enabled, since this value was not added by me (it already existed before my change in API/Server)

are we ok with not exposing virtio_scsi_multi_queues_enabled flag?
users indeed can do everything they want with virtio_scsi_multi_queues parameter but this will create mismatch between ansible-collection and python-sdk/API

what do you think?

Reply:
I think it's ok, we can also deprecate virtio_scsi_multi_queues_enabled if we didn't do that already 
CC @liranr23

Reply:
we didn't. we could do that since we introduced the number of multi queues. but in order to really do that some work need to be done on engine side (and api-model). basically it means to remove our consideration if multi queues are enabled or not. while considering the enabled to act as before.
i think we actually avoided it on the bug.

Reply:
> we didn't. we could do that since we introduced the number of multi queues. but in order to really do that some work need to be done on engine side (and api-model). basically it means to remove our consideration if multi queues are enabled or not. while considering the enabled to act as before. i think we actually avoided it on the bug.

didn't we do this already as part of https://gerrit.ovirt.org/c/ovirt-engine/+/115770/?

Reply:
i don't think so, as in BackendVmsResource we also have validation to have it enabled in order to set the number value.